lib/tasks/evm_test_helper.rb

```diff
@@ -17,4 +17,23 @@ def self.run_rake_via_shell(rake_command, env = {})
     _pid, status = Process.wait2(Kernel.spawn(env, cmd, :chdir => Rails.root))
     exit(status.exitstatus) if status.exitstatus != 0
   end
+
+  def self.vmdb_spec_directories
+    # TODO: Clean up this thing
+    #
+    # Within the spec directory, find:
+    #  * directories
+    #  * that aren't automation/migrations/replication (the excluded directories)
+    #  * that contain *_spec.rb files
+    #
+    # This is required because parallel_tests takes directories
+    # RSpec will sort out the parsing of _spec.rb's within them, too!
+    #
+    # Output: %w(./spec/controllers ./spec/helpers ./spec/initializers ..)
+    Dir.glob("./spec/*").select do |d|
+      File.directory?(d) &&
+        !EvmTestHelper::VMDB_EXCLUDED_SPEC_DIRECTORIES.include?(File.basename(d)) &&
```

Comment:
I believe we can do some Dir.glob to get all directories below spec that aren't the `EvmTestHelper::VMDB_EXCLUDED_SPEC_DIRECTORIES` directories but didn't know it off the top of my head.